### ISSUE TITLE
enhance import-name rule: allow different mappings

### ISSUE THREAD

**Issue by HamletDRC** [Microsoft/tslint-microsoft-contrib#137]
enhance import-name rule: allow different mappings

you should be able to add some options here to filter out false positives

There are the false positives we had: 

    import Backbone = require('backbone');
    import baseChartOptions = require('common/component/chart/options/BaseChartOptions');
    import classNames = require('classnames');
    import SentimentStatusType = require('Curated');
    import _ = require('underscore');

You should be able to configure the rule like this: 
```
    'import-name': [ true, {
            'backbone': 'Backbone',
            'react': 'React',
            'is-plain-object': 'isPlainObject',
            'BaseChartOptions': 'baseChartOptions'
        }]
```
